guides/05_custom-components/07_pdf-component-example.md

```diff
@@ -0,0 +1,672 @@
+# Case Study: A Component to Display PDFs
+
+Let's work through an example of building a custom gradio component for displaying PDF files.
+This component will come in handy for showcasing [document question answering](https://huggingface.co/models?pipeline_tag=document-question-answering&sort=trending) models, which typically work on PDF input.
+
+## Step 0: Prerequisites
+Make sure you have gradio 4.0 installed.
+As of the time of publication, the latest release is 4.1.1.
+Also, please read the [Five Minute Tour](./five-minute-guide) of custom components and the [Key Concepts](./key-component-concepts) guide before starting.
+
+
+## Step 1: Creating the custom component
+
+Navigate to a directory of your choosing and run the following command:
+
+```bash
+gradio cc create MyPDF
```

Comment:
Nit: Would `PDFViewer` be a better name for a component?
```suggestion
gradio cc create PDFViewer
```

Reply:
I just called it `PDF` which is what I originally used so that the screenshots are accurate and made a note about that.